numpy/lib/tests/test_function_base.py

```diff
@@ -2716,6 +2820,28 @@ def test_with_incorrect_minlength(self):
                             "must not be negative",
                             lambda: np.bincount(x, minlength=-1))
 
+        out = np.zeros(5, dtype=int)
+        assert_raises_regex(ValueError,
+                            "cannot be larger than the size of 'out'",
```

Comment:
I'd phrase this and the following as "out/initial must have size of at least ..."